### PR TITLE
Update nouislider

### PR DESCRIPTION
**Pull request by breavyn**
This fixes the issue of the slider handles sticking to the cursor after interaction has finished.

nouislider no longer uses jquery, so there are some code changes to integrate it.

### CODE REVIEW

**Comment by rburrow87** [comment]
This is with Ubuntu 24.10 using the deb artifact.

Mixer:
- If I enable override and set some value, when I switch to another tab and come back, it moves the slider back to 0 instead of the current value.

Adjustments:
- If I change from `Off` to `Mapped` and then set it to `Rate Profile Selection`, the slider to adjust the selection range cannot be dragged around. If I go to another tab and come back, it works as expected. Applies to `Profile Selection` and probably others too.

- The `Enable Channel` slider moves when you double click it when set to `ALWAYS`, and even if the adjustment is off entirely.

**Comment by rburrow87** [comment]
Those issues seem fixed to me! Checked in a Windows 11 VM and Ubuntu again.

However, I just realized motor override does NOT turn off anymore when you try to toggle it off and you have an override >0% set. I tried the last master branch CI build (ed1e978) and it works fine there, so I'm wondering if that's related to something in this PR?

File: src/css/tabs/adjustments.css [hunk 1]
```diff
@@ -210,37 +216,26 @@
...
+    padding-top: 4px;
 }
 
-.tab-adjustments .adjSliders .noUi-handle:before,
-.tab-adjustments .adjSliders .noUi-handle:after {
+.tab-adjustments .adjSliders .handle-icon:before,
+.tab-adjustments .adjSliders .handle-icon:after,
+.tab-adjustments .adjSliders .handle-icon:before,
+.tab-adjustments .adjSliders .handle-icon:after {
     height: 6px;
```
Comment (rotorflight) [inline src/css/tabs/adjustments.css:237]: Looks like a duplicate?

**Review by rotorflight — CHANGES_REQUESTED** [review]
Check the possible dup

**Review by rotorflight — APPROVED** [review]

**Pull request merged**